internal/service/vpc/resource_gcp_vpc_peering_connection.go

```diff
@@ -241,5 +247,25 @@ func copyGCPVPCPeeringConnectionPropertiesFromAPIResponseToTerraform(
 		return diag.FromErr(err)
 	}
 
+	var toProjectId string
+	var toVPCNetwork string
+
+	if len(*peeringConnection.StateInfo) > 0 {
```

Comment:
`*peeringConnection.StateInfo` is duplicated multiple times, please consider storing it in a variable